src/amber/controller/base.cr

```diff
@@ -13,11 +13,15 @@ module Amber::Controller
     protected getter raw_params : HTTP::Params
     protected getter context : HTTP::Server::Context
     protected getter params : Amber::Validators::Params
+    protected getter flash : Amber::Router::Flash::Hash
```

Comment:
I don't think we should be using a crystal lib class name it feels that we are overwriting the class behavior. What about `Amber::Pipe::FlashHash`?